_categories/models_import.md

```diff
@@ -29,10 +29,10 @@ subcategories:
         desc: 'Generate URDF (superset of SDFormat) and use it in Gazebo'
         star: false
 
-  - title: "Importing models from Google SketchUp"
-    items: 
-    - title: 'Importing Google 3D Warehouse model'
-      url: 'http://projects.csail.mit.edu/pr2/wiki/index.php?title=Importing_a_Google_3D_Warehouse_Model_%28.skp%29_into_Gazebo'
-      desc: 'Importing a .skp (Google warehouse format) into Gazebo'
-      star: true
+  # - title: "Importing models from Google SketchUp"
+  #   items: 
+  #   - title: 'Importing Google 3D Warehouse model'
+  #     url: 'http://projects.csail.mit.edu/pr2/wiki/index.php?title=Importing_a_Google_3D_Warehouse_Model_%28.skp%29_into_Gazebo'
```

Comment:
Can we use links to videos covering the respective content?

I found this youtube video link to be good for this index item. 

https://www.youtube.com/watch?v=aP4sDyrRzpU

Reply:
Good question. My personal opinion is that having a video tutorial could as good as having a text tutorial. So +1.